src/drivers/heater/heater.cpp

```diff
@@ -48,7 +48,7 @@
 #include <drivers/drv_hrt.h>
 #include <drivers/drv_io_heater.h>
 
-#if defined(BOARD_USES_PX4IO) and defined(PX4IO_HEATER_ENABLED)
+#if defined(BOARD_USES_PX4IO_VERSION) and defined(PX4IO_HEATER_ENABLED)
```

Comment:
I'd say this line can be left unchanged.  With your other changes, once configured, everything works on a Cube Black.  Nice work @AlexKlimaj !

Reply:
I changed this to reflect how BOARD_USES_PX4IO_VERSION is used in the rest of the codebase. BOARD_USES_PX4IO was only used in this driver.